### ISSUE TITLE
Breaking changes with release 2.5.0

### ISSUE THREAD

**Issue by fentas** [nhost/hasura-backend-plus#529]
Hey :wave: 
took me a little bit to figure this out.
I had / have a problem with release 2.5.0 especially through #507 

First, docu (and references like docker-compose) does not reflect that you now required to set env `DATABASE_URL`. 
HBP just fails with
```sh
yarn run v1.22.5
$ node -r ./dist/start.js
Applying migrations
(node:27) UnhandledPromiseRejectionWarning: Error: connect ECONNREFUSED 127.0.0.1:5432
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1146:16)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:27) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:27) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Also, if stated fresh it enforces the `public.user` table as it will migrate before hasura migrations.

Side question:
Is it required that user id is `uuid` or does it also work with `integer` ?

Thanks

**Comment by elitan** [comment]
We've added information about `DATABASE_URL` in the release not:

https://github.com/nhost/hasura-backend-plus/releases/tag/v2.5.0

We also have a PR that will show the error more clearly:
https://github.com/nhost/hasura-backend-plus/pull/525

> Also, if stated fresh it enforces the public.user table as it will migrate before hasura migrations.

Yes, is that a problem for you?

> Side question:
> Is it required that user id is `uuid` or does it also work with `integer`?

 I think `uuid` only. Generally, don't change any schema that was auto-applied by HBP.

**Comment by fentas** [comment]
> We've added information about DATABASE_URL in the release note:
Ah. Overlooked this. But as it was my first try with hbp I looked into documentation and used the docker-compose from master which will fail atm as latest is 2.5.0 and `DATABASE_URL` is missing. Had to look into source to find the problem.

> #525
:+1: top

> Yes, is that a problem for you?
I had already a user table within my hasura migrations which wasn't created anymore and failed the migration process itself.
One problem was also that the current schema had Integer as pid so after using the the user table with uuid all other (current relations) did not work.

I reworked them now as it was not a big schema for now. : ) But for an existing (bigger) project this could be problematic.
